app/components/sidebars/settings/settings_sidebar_base.js

```diff
@@ -49,7 +49,7 @@ export default class SettingsSidebarBase extends PureComponent {
     constructor(props) {
         super(props);
         this.state = {
-            showStatus: props.isCustomStatusEnabled,
+            showStatus: true,
```

Comment:
Why is this being set to true as default?

Reply:
Because even if the custom status is not enabled, it does not affect the use of `showStatus`.

If you see here,  https://github.com/mattermost/mattermost-mobile/blob/3d5f8fbf9bb13d47d31305c28b651f7b5122bbe5/app/components/sidebars/settings/settings_sidebar_base.js#L248 if the custom status is not enabled we are just returning null.